src/txn/txn_rollback_to_stable.c

```diff
@@ -560,13 +569,19 @@ __rollback_abort_newer_row_leaf(
      */
     WT_ROW_FOREACH (page, rip, i) {
         if ((upd = WT_ROW_UPDATE(page, rip)) != NULL)
-            __rollback_abort_newer_update(session, upd, rollback_timestamp);
+            __rollback_abort_newer_update(session, upd, rollback_timestamp, &stable_update_found);
+        else
+            /*
+             * Reset the flag to make sure that we don't reuse an earlier row flag result for the
+             * current row.
+             */
+            stable_update_found = false;
```

Comment:
Style wise I'd rather this was done unconditionally before the call above, rather than in an `else` block. I think that change means the comment isn't as necessary.

Reply:
Fixed.